pkg/apps/registry/instantiate/rest.go

```diff
@@ -50,14 +50,14 @@ func (s *REST) New() runtime.Object {
 }
 
 // Create instantiates a deployment config
-func (r *REST) Create(ctx apirequest.Context, obj runtime.Object, createValidation rest.ValidateObjectFunc, _ bool) (runtime.Object, error) {
+func (s *REST) Create(ctx apirequest.Context, obj runtime.Object, createValidation rest.ValidateObjectFunc, _ bool) (runtime.Object, error) {
```

Comment:
I'd rather change the other one to `r`, no idea what `s` stands for here; it's usually the first letter

Reply:
s as strategy